### ISSUE TITLE
[ENH] - Add stata in the stack using pystata

### ISSUE THREAD

**Issue by therimalaya** [jupyter/docker-stacks#1889]
### What docker image(s) is this feature applicable to?

datascience-notebook, new community stack

### What changes are you proposing?

- Add `pystata` package
- install `libncurses5` which is a dependency

### How does this affect the user?

- Stata is well-known statistical software used heavily in epidemiology and econometrics.
- Stata has now provided API to call its functions directly from python through `pystata`
- Users can point to their Stata installation and it will run smoothly in the python-jupyter notebook
- Adding this to the stack will help a lot of users

### Anything else?

**For more information:**

- https://www.stata.com/python/pystata/index.html
- https://www.stata.com/new-in-stata/jupyter-notebooks/

**Comment by mathbunnyru** [comment]
Hi @therimalaya!

Are there any reasons people can't just install `pystata` on top of our images [like this](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/recipes.html#using-mamba-install-or-pip-install-in-a-child-docker-image)?
Also, as far as I understand, `pystata` is an interface to `Stata`, which is not open-source.

If the installation is not easy, please, contribute a recipe to install pystata.
https://jupyter-docker-stacks.readthedocs.io/en/latest/using/recipes.html

**Comment by therimalaya** [comment]
@mathbunnyru 

Obviously, people can change the image and add the `pystata` and related libraries. That is what I have been doing. Since it is just adding one package in python and can help others like me, I added this issue as a feature request. 

Since Stata is not open source and I am not sure about the license of `pystata`, I now think that it would be a good idea to add a separate recipe rather than adding it to the data science stack. However, if `pystata` has a public license (like MIT) then I would definitely suggest this addition.

**Comment by mathbunnyru** [comment]
> it would be a good idea to add a separate recipe rather than adding it to the data science stack

100% agree.

> However, if pystata has a public license (like MIT) then I would definitely suggest this addition.

I think it's not a good idea to add a package, which will be useless without paid software.
If you want to use use pystata, it means you already have to install stata in our docker image and change it (or at least mount the stata volume, maybe), so you probably know what you're doing and it's not gonna be more difficult for you to install one package on top of the image you want to use.

**Comment by mathbunnyru** [comment]
@therimalaya please, feel free to create a PR for the recipe, I will be happy to merge the PR like this.
For now, closing this issue.
